src/Semantic/Graph.hs

```diff
@@ -301,36 +299,32 @@ parsePythonPackage parser project = do
       resMap <- Task.resolutionMap project
       pure (Package.fromModules (Data.Abstract.Evaluatable.name (projectName project)) modules resMap) -- TODO: Confirm this is the right `name`.
     PythonPackage.Packages dirs -> do
-      filteredBlobs <- for dirs $ \dir -> do
-        let packageDir = projectRootDir project </> unpack dir
-        let paths = filter ((packageDir `isPrefixOf`) . filePath) (projectFiles project)
-        traverse (readFile project) paths
+      let filteredBlobs = do
+            dir <- dirs
+            let packageDir = projectRootDir project </> unpack dir
+            filter ((packageDir `isPrefixOf`) . blobPath) (projectBlobs project)
```

Comment:
This is in the `[]` monad now.

Reply:
Idle speculation: this might express nicely as a list comprehension.